api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -765,6 +765,21 @@ message HttpConnectionManager {
   // If this config is set, the Proxy-Status HTTP response header field is
   // populated. By default, it is not.
   ProxyStatusConfig proxy_status_config = 49;
+
+  // Configuration options for Unified Header Validation (UHV).
+  // UHV is an extensible mechanism for checking validity of HTTP requests as well as providing
+  // normalization for request attributes, such as URI path.
+  // The default UHV checks the following:
+  // 1. H/1 header map validity according to `RFC 7230 section 3.2<https://tools.ietf.org/html/rfc7230#section-3.2>`_
+  // 2. Syntax of H/1 request target URI and response status
+  // 3. H/2 header map validity according to `RFC 7540 section 8.1.2<https://tools.ietf.org/html/rfc7540#section-8.1.2`_
+  // 4. Syntax of H/2 pseudo headers
+  // 5. Syntax of Content-Length and Transfer-Encoding
+  // 6. Validation of H/1 requests with both Content-Length and Transfer-Encoding headers
+  // 7. Normalization of the URI path
```

Comment:
HTTP/3 checks?

details on default normalization or reflink to where we configure details